### PR TITLE
api: fix discard action, default deposit endpoints prefixes

### PR DESCRIPTION
**Pull request by hachreak**
* Fix discard action in order to to avoid last record published
  changes.

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
@hachreak `git commit --amend` - Fix -> Fixes, `s/to to/to/`

File: invenio_deposit/api.py [hunk 1]
```diff
@@ -434,7 +434,7 @@ def discard(self, pid=None):
             before_record_update.send(self)
 
             _, record = self.fetch_published()
-            self.model.json = record.model.json
+            self.model.json = dict(record.model.json)
```
Comment (jirikuncar) [inline invenio_deposit/api.py]: `deepcopy`?

Comment (hachreak) [inline invenio_deposit/api.py]: done

**Comment by hachreak** [comment]
@jirikuncar commit message changed.
can we :shipit: ?

**Comment by jirikuncar** [comment]
LGTM

**Pull request merged**